app/javascript/react/components/CoffeeshopsShowContainer.js

```diff
@@ -25,14 +28,37 @@ const CoffeeshopsShowContainer = (props) => {
     }
   }
 
+  const fetchCoffeeshopReviews = async () => {
+    try {
+      const response = await fetch(`/api/v1/coffeeshops/${props.match.params.id}/reviews`)
+      if(!response.ok) {
+        const errorMessage = `${response.status}: ${response.statusText}`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const body = await response.json()
+      setReviews(body.reviews)
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
```

Comment:
you've done an excellent job in this PR. What I will ask though is that you make this component go from making two fetch requests on useEffect to one. 

This means that `fetchCoffeeshop` should grab your coffeeshop data AND all of the reviews for that coffee shop. A serializer can help you do this as needed

That would also mean not needing review_controller#index or the controller test